### PR TITLE
1.20

### PR DESCRIPTION
**Pull request by wagyourtail**
:sunglasses:

### CODE REVIEW

**Comment by wagyourtail** [comment]
ignore the failing test action. github's having issues with actions today

File: src/main/java/baritone/pathing/movement/MovementHelper.java [hunk 1]
```diff
@@ -511,14 +510,14 @@ static boolean canWalkOn(BlockStateInterface bsi, int x, int y, int z) {
...
-                && state.getMaterial() == Material.WATER
+                && state.getBlock() == Blocks.WATER
                 && ((Integer) state.getValue(LiquidBlock.LEVEL)) == 0;
     }
 
     static boolean canUseFrostWalker(IPlayerContext ctx, BlockPos pos) {
         BlockState state = BlockStateInterface.get(ctx, pos);
         return EnchantmentHelper.hasFrostWalker(ctx.player())
-                && state.getMaterial() == Material.WATER
+                && state.getBlock() == Blocks.WATER
```
Comment (ZacSharp) [inline src/main/java/baritone/pathing/movement/MovementHelper.java:520]: You made sure mc also changed the logic like this?

Comment (wagyourtail) [reply src/main/java/baritone/pathing/movement/MovementHelper.java:520]: they removed materials class entirely, not quite sure if there's a better way to handle that but that does appear to be how frost walker works

Comment (ZacSharp) [reply src/main/java/baritone/pathing/movement/MovementHelper.java:520]: If it's how frostwalker works then it's correct.

Comment (wagyourtail) [reply src/main/java/baritone/pathing/movement/MovementHelper.java:520]: let me check. 1 sec

Comment (wagyourtail) [reply src/main/java/baritone/pathing/movement/MovementHelper.java:520]: updating
image

**Comment by RemainingToast** [comment]
Based 😤

File: settings.gradle [hunk 2]
```diff
@@ -43,7 +43,8 @@ rootProject.name = 'baritone'
 
 include("tweaker")
 if (System.getProperty("Baritone.enabled_platforms") == null) {
-    System.setProperty("Baritone.enabled_platforms", "fabric,forge")
+//    System.setProperty("Baritone.enabled_platforms", "fabric,forge")
```
Comment (CDAGaming) [inline settings.gradle:46]: Forge is now available, no longer needed afaik.

**Comment by THEGODFATHERRX** [comment]
when are we going to merge and release?

**Comment by Natekalusa** [comment]
@wagyourtail do you have an eta on when this will come out?

**Comment by ZacSharp** [comment]
If you can't wait just use a CI build. It's basically the same but you don't need to wait for someone to put a fancy label on it.

**Comment by peeteer1245** [comment]
The Forge version of baritone also needs the `versionRange` modified to minecraft version `1.20.0`. Otherwise forge rejects the mod.

See [forge/src/main/resources/META-INF/mods.toml](https://github.com/wagyourtail/baritone/blob/b389b573e54537eb4dbda871d59fdbb865e2e0f5/forge/src/main/resources/META-INF/mods.toml)

**Comment by ST-DDT** [comment]
Here a summary on how to get the mod from CI:

grafik

Click on `Details`.

grafik

Click on `Summary`.

grafik

Click on `Artifacts` and the download should start.

**Comment by ST-DDT** [comment]
MC 1.20.1 got released

**Pull request merged**